src/components/TooltipTrigger.js

```diff
@@ -61,9 +73,10 @@ class Tooltip extends Component {
   }
 
   render() {
-    const { children, side } = this.props
+    const { children, side, type } = this.props
     const { target, tooltip, viewport } = this.state
-    const gap = 10
+    let gap = 10
+    if (type === 'light') { gap = 5 }
```

Comment:
nit: Because `let` creates a mutable variable, it's best to reserve it for cases where we actually need to create state. In this case we could do `const gap = type === 'light' ? 5 : 10`

The long-term benefit of being strict about this is that when you _do_ use a `let`, it's a strong signal to readers: "Hey, this is state, pay careful attention here"